go-apps/meepctl/cmd/deploy.go

```diff
@@ -171,13 +172,31 @@ func deployCore(cobraCmd *cobra.Command) {
 	for _, app := range deployData.coreApps {
 		chart := deployData.gitdir + "/" + utils.RepoCfg.GetString("repo.core.go-apps."+app+".chart")
 		codecov := utils.RepoCfg.GetBool("repo.core.go-apps." + app + ".codecov")
+		userFe := utils.RepoCfg.GetBool("repo.deployment.user.frontend")
+		userSwagger := utils.RepoCfg.GetBool("repo.deployment.user.swagger")
+		altServer := utils.RepoCfg.GetBool("repo.deployment.alt-server")
 		flags := deployRunScriptsAndGetFlags(app, chart, cobraCmd)
 
 		// Set core flags
 		coreFlags := utils.HelmFlags(flags, "--set", "image.repository="+deployData.registry+"/"+app)
 		coreFlags = utils.HelmFlags(coreFlags, "--set", "image.tag="+deployData.tag)
 		if deployData.codecov && codecov {
 			coreFlags = utils.HelmFlags(coreFlags, "--set", "codecov.enabled=true")
+			coreFlags = utils.HelmFlags(coreFlags, "--set", "codecov.location="+deployData.workdir+"/codecov/"+app)
+		}
+		if userFe {
+			coreFlags = utils.HelmFlags(coreFlags, "--set", "user.frontend.enabled=true")
+			coreFlags = utils.HelmFlags(coreFlags, "--set", "user.frontend.location="+deployData.workdir+"/user/frontend")
+		}
+		if userSwagger {
+			// deployment level flag - not all apps use it
+			coreFlags = utils.HelmFlags(coreFlags, "--set", "user.swagger.enabled=true")
+			coreFlags = utils.HelmFlags(coreFlags, "--set", "user.swagger.location="+deployData.workdir+"/user/swagger")
+		}
+		if altServer {
+			// deployment level flag - not all apps use it
+			coreFlags = utils.HelmFlags(coreFlags, "--set", "altService.enabled=true")
+			coreFlags = utils.HelmFlags(coreFlags, "--set", "altIngress.enabled=true")
```

Comment:
Should these be moved to the switch case for service-specific helm flags. E.g. only the platform-ctrl requires the FE & Swagger & alt-service flags. Also, only the platform-ctrl & mon-engine require alt-ingress flag.